recipes/bacnet-stack/all/conanfile.py

```diff
@@ -42,37 +32,33 @@ def configure(self):
         del self.settings.compiler.libcxx
         del self.settings.compiler.cppstd
```

Comment:
```suggestion
        self.settings.rm_safe("compiler.libcxx")
        self.settings.rm_safe("compiler.cppstd")
```
https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html#settings